### ISSUE TITLE
Not possible to open files inside .app folders (Applications)

### ISSUE THREAD

**Issue by nrlquaker** [coteditor/CotEditor#1156]
## Environment

- CotEditor: 4.0.1 (455)
- System: macOS Version 11.2.1 (Build 20D74)
- Language: English

## Short Description

Not possible to open files inside `.app` folders in `Open..` dialog.

## Steps to Reproduce the Issue

`File -> Open...` -> Try to open files inside .app folder. For example in `/Applications/`.

## Expected Result

It should be possible to open files inside applications.

**Comment by 1024jp** [comment]
Hi, thank you for the feedback and sorry for taking time to getting down.

The open dialog in CotEditor was designed to be able to see inside packages, such as .app, but only when "Show hidden files" checkbox is selected.
However, after receiving your feedback, I realized it doesn't work anymore in the latest environment. Thanks to make me realize it. I've just fixed it, and the change will be allied to the next version.
